doxia-modules/doxia-module-apt/src/test/java/org/apache/maven/doxia/module/apt/AptSinkTest.java

```diff
@@ -126,6 +126,11 @@ protected String getSection5Block(String title) {
         return EOL + StringUtils.repeat(AptMarkup.SECTION_TITLE_START_MARKUP, 4) + title + EOL + EOL + EOL;
     }
 
+    /** {@inheritDoc} */
+    protected String getSection6Block(String title) {
+        return EOL + StringUtils.repeat(AptMarkup.SECTION_TITLE_START_MARKUP, 4) + title + EOL + EOL + EOL;
```

Comment:
4, not 5?

Reply:
Yes, APT only supports up to level 5 and converts everything above to level  5

Reply:
This needs a comment

Reply:
Done